docs/source/definition-of-done.md

```diff
@@ -0,0 +1,12 @@
+# Definiton of done
+
+ * [ ] The acceptance criteria described in the Task is met
```

Comment:
```suggestion
 * [ ] The acceptance criteria described in the Jira Issue are met
```
1 criterion, 2+ criteria

Reply:
waiting for a decision regarding existency of definition of done